wiki/Tournaments/OWC/2013/en.md

```diff
@@ -3,426 +3,393 @@ tags:
-
-Mappools
+# osu! World Cup 2013
+
+![osu! World Cup 2013 logo](logo.png)
+
+The **osu! World Cup 2013** (***OWC 2013***) is a country-based osu! tournament hosted by the osu! team. It is the 3rd installment of the osu! World Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp  |
+|-----------------------:|-------------------------|
+| Registration Phase | 2013-10-15/2013-10-28 |
+| Drawings  | 2013-10-31 (16:00 UTC+0)|
+| Group Stage | 2013-11-08/2013-11-10 |
+| Round of 16 | 2013-11-16/2013-11-17 |
+| Quarterfinals  | 2013-11-23/2013-11-24 |
+| Semifinals  | 2013-11-30/2013-12-01 |
+| Finals  | 2013-12-07/2013-12-08 |
+
+## Prizes
+
+In every world cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. As these items may change with every installment of the World Cups, it is only possible to give an estimate prize list for each World Cup.
+
+| Place | Prize(s) |
+| ----- | -------- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months supporter tag, special profile badge, OWC trophy, [osu!tablet](https://osu.ppy.sh/forum/t/169139) |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 3 months supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month supporter tag |
+
+## Organization
+
+The osu! World Cup 2013 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Members |
+| -------- | ------- |
+| Management | ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [dkun](https://osu.ppy.sh/users/154400) |
+| Map Selectors | ![][flag_AR] [Darksonic](https://osu.ppy.sh/users/570042), ![][flag_AR] [Wishy](https://osu.ppy.sh/users/495477), ![][flag_IT] [Chewin](https://osu.ppy.sh/users/617323) |
+| Streamers | ![][flag_AU] [peppy](https://osu.ppy.sh/users/2), ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55), ![][flag_CA] [Nyquill](https://osu.ppy.sh/users/682935), ![][flag_US] [Makar](https://osu.ppy.sh/users/686389) |
+| Commentators | ![][flag_FR] [Mr Color](https://osu.ppy.sh/users/116078), ![][flag_US] [Agnes](https://osu.ppy.sh/users/136982), ![][flag_US] [ztrot](https://osu.ppy.sh/users/6347), ![][flag_NO] [kriers](https://osu.ppy.sh/users/333241), ![][flag_NO] [MillhioreF](https://osu.ppy.sh/users/941094), ![][flag_AU] [LaVolpe024](https://osu.ppy.sh/users/597796), ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) |
+| Statistician | ![][flag_PL] [Marcin](https://osu.ppy.sh/users/722665) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/forum/t/160181/start=0)
+- [Livestream](https://www.twitch.tv/osulive/profile/pastBroadcasts)
+- [Statistics Sheet](https://docs.google.com/spreadsheet/ccc?key=0AsjrK0nkPsOfdGZmZ2VKZ05KV1pjdUE5VlpHYVlwZWc&usp=drive_web#gid=17)
+
+-------------------------------------------
+
+## Participants
```

Comment:
I would change order here, first the bracket, then the seeds and finally the participants.